File: ruby_small_problems/easy-2/03-tip-calc.rb


puts "What is the bill?"
bill = gets.to_f

puts "What is the tip percentage?"
tip_percentage = gets.to_f

tip = bill * tip_percentage / 100.0
tip.round(2)

puts format("The tip is $%.2f", tip)
puts format("The total is $%.2f", tip + bill)

File: python/small_problems/easy_2/03_tip_calc.py
# Create a simple tip calculator. The program should prompt for a bill amount
# and a tip rate. The program must compute the tip and then display both the
# tip and the total amount of the bill.

print("What is the bill?")
bill = float(input())

print("What is the tip percentage?")
percentage = float(input()) / 100

tip = round(bill * percentage, 2)
total = tip + bill

print(f"The tip is: {tip}")
print(f"The total is: {total}")

